Bottom: 55d747949e52937b82271d649e11920b4b534b80
Top:    a38f4698dc5dbb97b7ccef09ef3a65cb2cc4070d
Author: Jason Wang <jasowang@redhat.com>
Date:   2015-01-20 13:45:15 +0800

vhost: let vhost_signal() returns whether or not

Signed-off-by: Jason Wang <jasowang@redhat.com>


---

diff --git a/drivers/vhost/vhost.c b/drivers/vhost/vhost.c
index e75c2e2..4d6ac1a 100644
--- a/drivers/vhost/vhost.c
+++ b/drivers/vhost/vhost.c
@@ -1532,9 +1532,6 @@ void vhost_signal(struct vhost_dev *dev, struct vhost_virtqueue *vq)
 	int left;
 	ktime_t now;
 
-	if (!vq->call_ctx)
-		return;
-
 	if (can_coalesce) {
 		now = ktime_get();
 		left = vq->coalesce_usecs -
